### DISCUSSION TITLE
Quick ideas for the abyss

### DISCUSSION THREAD

**Discussion by GaroxLeChatRusse** [FakeFishGames/Barotrauma#5426] [Ideas and suggestions]
So right here: https://barotraumagame.com/gameplay-features/here-be-dragons-reintroducing-abyss-creatures/, you said this "t else do you think should be found in the abyss? Let us know". 

Well, what about specific structures/caves for the abyss, like really old wrecks or some jovian separatist's weapon caches. 

Another idea: Maybe you could add something like a "fog" ? Since there's a lot of volcanic activity and this is the bottom of the Ocean it could make sense and it could make the abyss truly scary. 

Last idea: Add the Mantis at the bottom of the ocean. Please !

**Comment by MartenKahr** [comment]
Something that immediately came to my mind when I saw that question, was the idea of seafloor alien ruins. Considerably more extensive and dangerous than the ones found up above. Perhaps even big enough that you need to carefully maneuver your sub into an alien sub-pen and connect to a docking hatch, leading to an on-foot exploration session where you can't easily work your way around obstacles by cutting open exterior walls to flood the room and swim around them.
